.devops/azure-templates/deploy-client-sdk.yml

```
@@ -0,0 +1,71 @@
+# Azure DevOps pipeline template to release Client SDK to NPM
+
+parameters:
+
+  - name: 'dependsOn'
+    type: string
+    default: 'No_Name'
```

We can try to improve this a little by passing an array instead of a single stage, so that we can have more flexibility for the user and mirror perfectly the actual stage behaviour. We can have `[]` as default value. 
Usage would be
```yaml
  - template: azure-templates/deploy-client-sdk.yml
    parameters:
      dependsOn: 
       - Release
```

Try:
```suggestion
  - name: 'dependsOn'
    type: string
    default: []
```

---

done